app/src/main/java/com/ggetters/app/data/repository/user/CombinedUserRepository.kt

```
@@ -26,61 +26,124 @@ class CombinedUserRepository @Inject constructor(
     override suspend fun insertRemote(user: User) = online.insertRemote(user)
 
     override fun hydrateForTeam(id: String) {
-        // Run sync in the background so it won't block UI
         kotlinx.coroutines.CoroutineScope(kotlinx.coroutines.Dispatchers.IO).launch {
+            val trace = FirebasePerformance.getInstance().newTrace("userrepo_hydrateForTeam")
+            trace.start()
             try {
                 Clogger.d("UserRepo", "Hydrating users for team $id …")
                 sync()
+                trace.putMetric("hydrated", 1)
             } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is too generic. Prefer catching specific exceptions to the case that is currently handled.

(detekt.exceptions.TooGenericExceptionCaught)